src/distributions/gamma.rs

```diff
@@ -304,27 +304,32 @@ mod test {
     use distributions::Distribution;
     use super::{Beta, ChiSquared, StudentT, FisherF};
 
+    #[cfg(not(miri))] // Miri is too slow
+    const N: u32 = 1000;
+    #[cfg(miri)]
+    const N: u32 = 100;
```

Comment:
These tests don't do a whole lot, so reducing the count to 100 everywhere is fine.